MSLaunches.Backend/MSLaunches.Api/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -41,14 +41,13 @@ public ErrorHandlingMiddleware(RequestDelegate next)
             }
         }
 
-        private static Task HandleExceptionAsync(HttpContext context, Exception exception)
+        private Task HandleExceptionAsync(HttpContext context, Exception exception)
         {
             var code = HttpStatusCode.InternalServerError; // 500 if unexpected
 
             if (exception is UnauthorizedException) code = HttpStatusCode.Unauthorized;
-            // else if (exception is SomeOtherException) code = HttpStatusCode.RequestTimeout;
-            // else if (exception is SomeOtherException2) code = HttpStatusCode.BadRequest;
-            context.Response.Clear();
+            else if (exception is ValidationException) code = (HttpStatusCode)422;
+            else if (exception is NotFoundException) code = HttpStatusCode.NotFound;
```

Comment:
love these midlewares

Reply:
the First person i met that really likes, like me, catching std on APIs.